primeqa/pipelines/components/reader/extractive.py

```diff
@@ -28,14 +29,20 @@ class ExtractiveReader(ReaderComponent):
         scorer_type (str, optional): Scoring algorithm. Defaults to "weighted_sum_target_type_and_score_diff".
         min_score_threshold: (float, optional): Minimum score threshold. Defaults to None.
 
+    Important:
+        1. Each field has metadata property which can carry additional information for other downstream usages.
+        2. Two special keys (api_support and exclude_from_hash) are defined in "metadata" property.
+            a. api_support (bool, optional): If set to True, that parameter is exposed via service layer. Defaults to False.
+            b. exclude_from_hash (bool,optional): If set to True, that parameter is not considered while building the hash representation for the object. Defaults to False.
+
 
     Returns:
         _type_: _description_
     """
 
     model: str = field(
         default="PrimeQA/tydiqa-primary-task-xlm-roberta-large",
```

Comment:
can we have @rongzhangibm 's model uploaded and point to that one instead? That model is at least a couple of points better.